docs/reference/query-dsl/query-string-syntax.asciidoc

```diff
@@ -30,6 +30,11 @@ You can specify fields to search in the query syntax:
 
     author:"John Smith"
 
+* where the `first name` field contains `Alice` (note how we need to escape
```

Comment:
It seems like your example assumes that the query string is supplied through the URL (and therefore spaces get encoded as `+` or `%20). It'd be best if these examples used unencoded strings:

```
first\ name:Alice
```

Then users can modify the example depending on how they are passing the query string to Elasticsearch (this will differ based on whether they are using Kibana, passing it through the request body, or through the URL).

Reply:
Thank you, I've made the suggested edit. I wasn't aware that the query string syntax could be used outside of a URL! I should have noticed I wasn't matching the other examples though.